packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -313,12 +313,14 @@ export class MirrorNodeClient {
     path: string,
     pathLabel: string,
     method: REQUEST_METHODS,
+    requestIdPrefix: string,
     data?: any,
-    requestIdPrefix?: string,
     retries?: number,
   ): Promise<T | null> {
     const start = Date.now();
     // extract request id from prefix and remove trailing ']' character
+    this.logger.debug(requestIdPrefix);
+    this.logger.debug(`${method} + ${path} + ${requestIdPrefix}`);
     const requestId =
       requestIdPrefix
         ?.split(MirrorNodeClient.REQUEST_PREFIX_SEPARATOR)[1]
```

Comment:
If we add a field which contains the raw `requestId` inside `requestDetails`, this logic won't be needed anymore

Reply:
Is it worth it to add requestId and requestIdPrefix everywhere, just so we remove this. Or if you mean to pass raw requestId everywhere, then we need to format it in every method

Reply:
We can only add the raw `requestId` to the `RequestDetails` object and deduct the prefixed version dynamically as I suggested in https://github.com/hashgraph/hedera-json-rpc-relay/pull/2939#discussion_r1751687265.